build/test/OverlaySdkOnLKG.targets

```diff
@@ -14,6 +14,6 @@
   </Target>
 
   <Target Name="PublishRedist">
-    <Exec Command="$(DotNetExe) publish $(RepoRoot)/src/redist/redist.csproj --no-build --output $(CliOutputDirectory)" WorkingDirectory="$(RepoRoot)/src/redist" />
+    <Exec Command="$(DotNetExe) publish $(RepoRoot)/src/redist/redist.csproj --output $(CliOutputDirectory)" WorkingDirectory="$(RepoRoot)/src/redist" />
```

Comment:
Change this to an `MSBuild` task invocation if possible

Reply:
Will do.

Reply:
Actually, I did that already, but you are reviewing by commit it seems, so, in this commit this hadn't happened yet.